README.md

```diff
@@ -73,7 +68,18 @@ SSL:
   --cacert CACERT       Path to CA certificate to use for SSL certificate
                         verification
 
+
+```
+
+You can filter MRs/PRs based on their relative age
+```
+review-rot --age older 5d 10h
+```
+outputs MRs/PRs which ale older than 5 days and 10 hours,
+```
+review-rot --age newer 5d 10h
 ```
+outputs MRs/PRs which are newer than 5 days and 10 hours
```

Comment:
Same here, submitted in the last 5 days and 10 hours

Reply:
fixed